stdlib/package.py

```diff
@@ -41,8 +42,7 @@ def __init__(
     ):
         build = stdlib.build.current_build()
 
-        import core.args
-        self.repository = repository if repository is not None else core.args.get_args().repository
+        self.repository = repository if repository is not None else core.config.get_config()['global']['target']
```

Comment:
```suggestion
        self.repository = repository or core.config.get_config()['global']['target']
```
I think the above works in Python ;)